### ISSUE TITLE
Still Maintained/Active?

### ISSUE THREAD

**Issue by zquestz** [bil-elmoussaoui/Hardcode-Tray#778]
This project has not been updated in two years with quite a few unmerged PRs. Is this project still actively maintained? Should we still be reporting issues and putting up PRs?

@bilelmoussaoui is this still being supported?

**Comment by SebaRana678** [comment]
No, it's not still active. Perhaps we could keep the paths of the icons we want to replace up to date using this method, but we'd have to do everything manually. The Electron script also doesn't work as it should in many applications; I do it manually with "asar." I can edit the .json files to keep them relatively up-to-date on my system, but I can't automate the .asar files I configure manually since I have no programming knowledge. It's a real shame.

**Comment by bilelmoussaoui** [comment]
The project is basically dead because I no longer personally use it. If someone wants to step up as a maintainer, i can give them access to it.
